<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesTraktor</class>
 <widget class="QWidget" name="PreferencesTraktor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>239</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="checkBoxUseTraktorKey">
     <property name="text">
      <string>Read 'Key' field instead of 'Key Text' when reading from Traktor</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxMergeGridMarkers">
     <property name="text">
      <string>Merge grid markers and hotcues found at the same position when writing to Traktor</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxImportComments2">
     <property name="text">
      <string>Use the 'Comments2' field as 'Grouping' when reading/writing from/to Traktor</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
